### DISCUSSION TITLE
Micronaut-GraalVM YamlPropertySourceLoader cannot find resources in a transitive dependency

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#9829] [Q&A] [answered]
Hi All,
In my ApplicationContextConfigurer I am using following. And myapp,.yml comes from a transitive dependency.(in a different jar's src/main/resources/myapp.yml). in addition I have application.yml also in applications's resources folder of course. 

```
Optional<PropertySource> ps = new YamlPropertySourceLoader().load("myapp",new DefaultClassPathResourceLoader(this.getClass().getClassLoader()));
builder.propertySources(ps.get())
```

in a JVM this works fine and my config get picked up. But when compiled into native image and run, it cannot find myapp resource which is part of a  transitive dependency jar.

In my native maven plugin, resources-config.json is as follows.

```
{
  "resources": {
    "includes": [
      {"pattern": ".*\\.yml$"}
    ]
  }
}
```

Any idea?

**Comment by subash89** [comment]
Appreciate any input.

**Comment by msupic** [comment]
Could you please create a sample app with a reproducer.

**Comment by subash89** [comment, marked as answer]
Turns out it was some Jackson usage which not supports in native image which caused the behavior for my app. Resources in transitive dependencies are actually getting picked from this resource-config.json as expected.
